src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -912,20 +914,21 @@ ReactDOMComponent.Mixin = {
           deleteListener(this, propKey);
         }
       } else if (isCustomComponent(this._tag, nextProps)) {
-        if (propKey === CHILDREN) {
-          nextProp = null;
```

Comment:
I was not sure what is the point of setting `nextProp` to `null` here. It caused `removeAttribute()` call but it’s not like this attribute existed in the first place? Since this didn’t break any tests, I just put it behind an `if`.